src/main/java/com/yelp/nrtsearch/server/luceneserver/highlights/NRTFastVectorHighlighter.java

```diff
@@ -142,11 +142,10 @@ public String[] getHighlights(
           DEFAULT_ENCODER);
     } catch (RuntimeException runtimeException) {
       logger.warn(
-          String.format(
-              "FVH failed creating fragments for the luceneDocId %d, the highlight query is %s and the exception is: %s",
+              "FVH failed creating fragments for the luceneDocId: {}, highlight query: {}, exception: {}",
               leafDocId + hitLeaf.docBase,
               settings.getHighlightQuery(),
-              runtimeException.getMessage()));
+              runtimeException.getMessage());
```

Comment:
Can you please add the exception as well like I had added in my previous comment?
Also can you please test it locally and verify that the log works as expected now? You can add the logged message in the PR description.

Reply:
It's hard to reproduce the error unless we have a buggy analyzer. I created a dummy exception for the test purpose which will serve the test purpose.